app/models/medication.rb

```diff
@@ -0,0 +1,6 @@
+class Medication < ApplicationRecord
+  validates :brand_name, :generic_name, :med_id, presence: true
+
+  has_many :user_medications
+  has_many :users, through: :user_medications
+end
```

Comment:
We may want to add a `has_many` relationship with `symptoms`.

Reply:
Like a : 
```
has_many :medication_symptoms
has_many :symptoms, through: :medication_symptoms
```

Reply:
Yes. I don't think we need it right now but it could be useful in the future.

Reply:
Have we decided what endpoint we're using for the medication search and confirmed that it actually gives us all the info we are intending to record here?

Reply:
We can use the NDC endpoint for the medication search:

https://api.fda.gov/drug/ndc.json?search=brand_name:zoloft&limit=1

Then, we can use either the brand name or generic name from that search to search for the adverse reactions:

https://api.fda.gov/drug/label.json?search=adverse_reactions:zoloft&limit=1

This _does_ mean that we should maybe be just using `brand_name` or `generic_name` to get the adverse reaction information as opposed to using a medication ID.